src/main/java/com/cx/plugin/cli/utils/CxConfigHelper.java

```diff
@@ -500,9 +500,30 @@ private void setScaSpecificConfig(CxScanConfig scanConfig) throws CLIParsingExce
 
         sca.setConfigFilePaths(Arrays.asList(getOptionalParam(SCA_CONFIG_FILE, "")));
         sca.setSastProjectId(getOptionalParam(SAST_PROJECT_ID, ""));
-        sca.setSastServerUrl(getOptionalParam(SERVER_URL, ""));
-        sca.setSastUsername(getOptionalParam(USER_NAME, ""));
-        sca.setSastPassword(getOptionalParam(USER_PASSWORD, ""));
+       // When SAST & SCA together

```

Comment:
Move this code in a private method and try to optimize if..else . SonarQube raises this issue related to "Cognitive Complexity" if the code has lot of if elses.